xray/segment.go

```diff
@@ -61,6 +62,13 @@ func BeginSegment(ctx context.Context, name string) (context.Context, *Segment)
 }
 
 func BeginSegmentWithSampling(ctx context.Context, name string, r *http.Request, traceHeader *header.Header) (context.Context, *Segment) {
+	// If SDK is disabled then return with an empty segment
+	sdkDisable := Disabled()
```

Comment:
Is there a reason to have this local variable instead of just `if Disabled()`?

Reply:
Makse sense! will modify it